libcloud/dns/drivers/cloudflare.py

```diff
@@ -146,27 +147,37 @@ def parse_error(self):
             raise exception_class(**kwargs)
 
 
-class CloudFlareDNSConnection(ConnectionUserAndKey):
+class BaseDNSConnection:
```

Comment:
Minor style thing - we tend to explicitly inherit from object everywhere.

Reply:
Fixed in 1d209ed8.